### PR TITLE
18 invoice jbuilder record endpoint

### PR DESCRIPTION
**Pull request by seeker105**
Setup and begin to implement testing the API using RSpec. Create test for the Items/FindController#show action

*Install FactoryGirl in the project.
*Create a factory file for the models.

### CODE REVIEW

File: spec/controllers/api/v1/items_find_controller_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Items::FindController, type: :controller do
+  render_views
```
Comment (seeker105) [inline spec/controllers/api/v1/items_find_controller_show_spec.rb]: Vital. Without this line `response.body` comes back as an empty string.

File: spec/controllers/api/v1/items_find_controller_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Items::FindController, type: :controller do
```
Comment (seeker105) [inline spec/controllers/api/v1/items_find_controller_show_spec.rb]: You have to specify the `type: :controller` to tell it what kind of test this is.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -20,7 +20,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```
Comment (seeker105) [inline spec/rails_helper.rb]: This tells Rails to pull in the files in the 'spec/support' folder.

File: spec/support/factory_girl.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+require 'factory_girl_rails'
```
Comment (seeker105) [inline spec/support/factory_girl.rb]: This is a key piece I was missing when trying to get FactoryGirl working.

**Comment by robbiejaeger** [comment]
Thanks for all of the testing notes!

**Pull request merged**